src/download.cpp

```diff
@@ -234,8 +234,8 @@ void DownloadFile(const std::string& url, IDownloadSink *sink, Thread *onThread,
                          (
                              inet,
                              url.c_str(),
-                             NULL, // lpszHeaders
-                             -1,   // dwHeadersLength
+							 headers.c_str(), // lpszHeaders
```

Comment:
As you can see yourself here, your changes break formatting all over the place. Please respect existing code's indentation etc. style, including the use of tabs vs. spaces and the amount of indentation.